internal/component/database_observability/postgres/collector/dsn.go

```diff
@@ -0,0 +1,30 @@
+package collector
```

Comment:
These first few files include changes moving some database connectivity functions and tests from explain_plan.go to dsn.go.